### ISSUE TITLE
Standardized variants for Manichaean

### ISSUE THREAD

**Issue by brawer** [googlei18n/noto-fonts#738]
When Noto adds support for the Manichaean script, which was added to Unicode 7.0, it should not forget to support the following [standardized variants](http://www.unicode.org/Public/UCD/latest/ucd/StandardizedVariants.txt):

10AC5 FE00; alternate form; isolate # MANICHAEAN LETTER DALETH
10AC6 FE00; alternate form; isolate # MANICHAEAN LETTER HE
10AD6 FE00; alternate form; isolate final # MANICHAEAN LETTER MEM
10AD7 FE00; alternate form; isolate # MANICHAEAN LETTER NUN
10AE1 FE00; alternate form; isolate # MANICHAEAN LETTER RESH

**Comment by KatMomoi** [comment]
This looks like a product decision bug. Let me assign this to xiangye. Does the Noto project plan to support all historical scripts of which Manichaean is one?

**Comment by brawer** [comment]
Doesn’t Noto plan to support all of Unicode?

**Comment by KatMomoi** [comment]
I stand corrected -- the Noto project is already supporting a number of historical scripts like Avestan, Brahmi, and others.  And if it plans to support all Unicode scripts -- extinct or alive, then Manichaean is definitely to be included.
Then, this is a reminder bug for specs of such a font. I guess it still is appropriate to have this assigned to xiangye?

**Comment by xiangyexiao** [comment]
yep. It is correct to assign this kind of bugs to me. I will reassign it to MTI as a reminder.

**Comment by xiangyexiao** [comment]
Sue, this is a reminder bug to implement the five standardized variants of Manichaean when you start to develop font for it.
